### PR TITLE
CORTX-28518_Alex_Scan: Refactoring: Addressed Inconsiderate words rep…

### PR DESCRIPTION
**Pull request by kailashyadav11**
…orted by Alex scan report

Signed-off-by: Kailash Yadav <kailash.yadav@seagate.com>

# Problem Statement
- Use of insensitive, inconsiderate  words are harmful such as gender favoring, polarizing, race related, religion inconsiderate, or other unequal phrasing in text.

# Design

-  [For Feature, Post the link for design
](https://jts.seagate.com/browse/CORTX-28518)
# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

-----
[View rendered test/setup/README.md](https://github.com/Seagate/cortx-prvsnr/blob/CORTX-28518_Alex_Scan/test/setup/README.md)

### CODE REVIEW

**Comment by kailashyadav11** [comment]
retest this please

File: test/deploy/kubernetes/README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ kubectl logs <pod-name>
 ```
 <br>
 
-## Destroy ALL CORTX PODs **(On Master Node)**
+## Delete ALL CORTX PODs **(On Master Node)**
```
[the remark this answers is not in the document]
Comment (kailashyadav11) [reply test/deploy/kubernetes/README.md]: done

**Comment by kailashyadav11** [comment]
retest  this please

**Review by tanujashinde0405 — APPROVED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**